packages/kbn-dev-utils/src/code_owners/owners_source_of_truth.ts

```diff
@@ -0,0 +1,300 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export const sourceOfTruth = [
+  {
+    title: 'App',
+    githubHandle: 'elastic/kibana-app',
+    pathPatterns: [
+      '/x-pack/plugins/dashboard_enhanced/',
```

Comment:
I don't think this layout by team works for both purposes of CODEOWNERS and code coverage.  The common key between those 2 sets of data is the path, not the team.

I think the key to this structure should be the path, with a field for `codeOwners: [], codeCoverage: []`

If you don't want to require team review on PRs for a path, leave that codeOwners array empty (you may very likely still have team(s) in the codeCoverage array.

The CODEOWNERS file could still be output grouped by the team(s).

Reply:
We could just call the codeowners code from the code coverage ingestion. 
That would give us the map object, with each key being a path.
Then we could enrich that map with a list of paths and teams that lives within the code coverage module.

Or, we could simply enrich the codeowners source of truth, but that could be mixing concerns.